data/index.rst

```diff
@@ -350,7 +413,7 @@ Summit mounts a POSIX-based IBM Spectrum Scale parallel filesystem called Alpine
    Figure 1. An example of the NDS servers on Summit
 
 =====================================
-Performance under non-ideal workloads
+Alpine Performance under non-ideal workloads
 =====================================
```

Comment:
The `=` signs need to span the length of the title (lines 415 and 417 need to have more `=` signs)